index.Rmd

```diff
@@ -1,105 +1,228 @@
-                size = runif(n, 5, 20),
-                category = factor(
-                  sample(letters[1:5], n, replace = TRUE)
-                  ),
-                value = rnorm(n))
+
+#For Mars
+nlayers(image2) #This give the number of bands use
+crs(image2) #the coordinate system the imagery is projected in, and the resolution (or grid cell size) of the raster.
+res #resolution of the images
 ```
 
-```{r, results='asis'}
-data %>% 
-  slice(1:10) %>% #show only 1:n rows
-  kable(digits=2,align="c")%>% #make table and round to two digits
-  kable_styling(bootstrap_options = 
-                  c("striped", "hover", "condensed", "responsive")) #apply other formatting
+##Results1
+Now that we know a little more about the imagery we are using, let plot it. Since the image is a multi-band raster,
+we use the plotRGB function from the raster package, which allows us to specify which bands should be visualized.
+
+
+###FALSE COLOR PLOT.The false color composite uses NIR (5) for red, red (4) for green, and green (3) for blue.
+
+
+```{r}
+
+par(col.axis="white",col.lab="white",tck=0)
+plotRGB(image2, r = 5, g = 4, b = 3, axes = TRUE, stretch = "lin", main = "False Color Composite")
+box(col="white")
 ```
+###Indices of WATER
 
-Add any additional processing steps here.
+We can use Modified Normalized Difference Water Index [MNDWI=(Green-MIR)/ (Green+MIR)] to separate water from land. This idea is from Reference=https://www.researchgate.net/post/What_is_the_best_band_combination_for_highlighting_the_water_and_soil_together_in_LANDSAT_image2
+The Author of the reference is Saygin Abdikan and is currently working at the Department of Geomatics Engineering, Hacettepe University. The Author researchGate is https://www.researchgate.net/profile/Saygin_Abdikan
 
-# Results
+The formula is as below:
+[MNDWI=(Green-MIR)/ (Green+MIR)]
+MNDWI <- (image[[2]] - image[[3]])/(image[[2]] + image[[3]]).
 
-[~200 words]
+To plot the results with ggplot, we convert the raster into a data frame and use geom_tile.
+The plot generated by the codes below show where are located water/ water ice/ volatiles materials the most and we have then know where to land the rovers and start the mining process.
 
-Tables and figures (maps and other graphics) are carefully planned to convey the results of your analysis. Intense exploration and evidence of many trials and failures. The author looked at the data in many different ways before coming to the final presentation of the data.
+Band4, Band5, and Band6 are use for Earth datas where Band4 represent the Green band in our case and Band6 represent the MIR band.
 
-Show tables, plots, etc. and describe them.
+For Mars,the formula change little bit because all bands are collected and stored in the same file in Mars Odysee website. False color or black and white could reveal the distribution of water. However, we use the formula:
+
+MNDWI <- (band4)/(1/band9).
+This formula is derived after multiple simulations and observations
```

Comment:
what do you mean by 'multiple simulations and observations'?  Please specify and preferably include the code here.